master/buildbot/reporters/bitbucket.py

```diff
@@ -112,9 +113,16 @@ def sendMessage(self, reports):
         props = Properties.fromDict(build['properties'])
         props.master = self.master
 
+        def key_hash(key):
+            sha_obj = hashlib.sha1()
+            sha_obj.update(key.encode('utf-8'))
+            return sha_obj.hexdigest()
+
+        status_key = (yield props.render(self.status_key))
```

Comment:
There's extra pair of parentheses. Better use `status_key = yield props.render(self.status_key)`. In the line below the parentheses were required because the expression was part of a dict literal.

Reply:
Thanks!